EIPS/eip-5615.md

```diff
@@ -4,10 +4,11 @@ title: EIP-1155 Supply Extension
 description: A simple mechanism to fetch token supply data from EIP-1155 tokens
 author: Pandapip1 (@Pandapip1)
 discussions-to: https://ethereum-magicians.org/t/eip-5615-eip-1155-supply-extension/10732
-status: Review
+status: Last Call
+last-call-deadline: 2023-01-01
 type: Standards Track
 category: ERC
-created: 2022-09-07
+created: 2023-03-08
```

Comment:
The `created` field should be immutable.

Reply:
Oops, I think I updated the wrong field lol.

Reply:
Also, the created field shouldn't be in the future.